demo/src/lib/stackblitz/svelte-tailwind/index.html

```diff
@@ -0,0 +1,12 @@
+<!doctype html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<title>AgnosUI svelte stackblitz demo</title>
+	</head>
+	<body style="margin: 1rem; min-height: 100vh!">
```

Comment:
for tailwind, we can do the same, but the classes would be "m-4 h-dvh" (margin 1rem, height 100dvh) but we need to update the tailwind.config.js with the updated content: content: ['./src/**/*.{html,js,svelte,ts,tsx}', './index.html'],

Reply:
or min-h-dvh (min height)